pkgs/development/python-modules/diff_cover/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, jinja2, jinja2_pluralize, pygments,
+  six, inflect, mock, nose, coverage, pycodestyle, flake8, pyflakes, git,
+  pylint, pydocstyle }:
+
+buildPythonPackage rec {
+  pname = "diff_cover";
+  version = "1.0.2";
+
+  LC_ALL = "en_US.UTF-8";
+
+  src = fetchFromGitHub {
+    owner = "Bachmann1234";
+    repo = "diff-cover";
+    rev = "85c30959c8ed2aa3848f400095a2418f15bb7777";
```

Comment:
you can refer to a tag here

Reply:
If I refer to the last tag, tests will fail. Maybe it is better refer to this commit?

Reply:
https://github.com/NixOS/nixpkgs/pull/33715#discussion_r160911681